### PR TITLE
Adding new templates

### PR DESCRIPTION
**Pull request by SarahJohnsonONS**
Adding new templates

### CODE REVIEW

File: src/csvcubed/readers/cubeconfig/v1_0/templates/mixed-period.json [hunk 1]
```diff
@@ -0,0 +1,7 @@
+{
+    "type": "dimension",
+    "from_existing": "http://purl.org/linked-data/sdmx/2009/dimension#refPeriod",
+    "label": "Mixed Time Period",
+    "description": "Template allowing the use of different time periods within one dimension, e.g. 2020-Q4 (quarter), 2021-06 (month), 2019-08-06 (day)",
```
Comment (robons) [inline src/csvcubed/readers/cubeconfig/v1_0/templates/mixed-period.json:5]: The `description` describes the dimension, not the template. So I'd reword it if you could.

File: src/csvcubed/schema/cube-config/v1_4/schema.json [hunk 2]
```diff
@@ -677,7 +677,10 @@
                         "government-half-year",
                         "government-quarter",
                         "government-week",
-                        "gregorian-instant"
+                        "mixed-period",
```
Comment (robons) [inline src/csvcubed/schema/cube-config/v1_4/schema.json:680]: I'd forgotten about this, nice catch.

**Review by robons — APPROVED** [review]
Fundamentally looks good, but take another look at the `description` field before merging.

File: src/csvcubed/readers/cubeconfig/v1_0/templates/mixed-period.json (hunk 1 above)
Comment (SarahJohnsonONS) [reply src/csvcubed/readers/cubeconfig/v1_0/templates/mixed-period.json:5]: Thanks, amended
